Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Model/Building/SqlStorageTypeInformationProvider.cs

```diff
@@ -226,6 +231,28 @@ public IStorageTypeInformation GetStorageType (object? value)
       return null;
     }
 
+    private bool TryGetDateTimeStorageTypeInformation (PropertyDefinition propertyDefinition, bool isNullableInDatabase, Type dotNetType, [NotNullWhen(true)] out StorageTypeInformation? dateTimeStorageTypeInformation)
```

Comment:
We're using nullable return values everywhere else. For consitency, we should continue this pattern with new members in this class. => pelase switch to `GetDateTimeStorageType` returning a nullable StorageTypeInformation. I also dropped the `Information` for naming consistnecy